lib/js-file.js

```diff
@@ -425,12 +425,10 @@ JsFile.prototype = {
      * @returns {Object|undefined}
      */
     getCommentAfterToken: function(token) {
```

Comment:
This likely becomes deprecated now, as its usage is superceded by getNextToken(true). We can fix that later though.